public.yaml

```diff
@@ -245,6 +245,26 @@ spec:
       value: saltapi
     - name: VELUM_SALT_PASSWORD_FILE
       value: /var/lib/misc/infra-secrets/saltapi-password
+    - name: LDAP_HOST
+      value: "127.0.0.1"
+    - name: LDAP_PORT
+      value: "389"
+    - name: LDAP_GROUP_BASE_DN
+      value: ou=Groups,dc=infra,dc=caasp,dc=local
+    - name: LDAP_PEOPLE_BASE_DN
+      value: ou=People,dc=infra,dc=caasp,dc=local
+    - name: LDAP_BIND_DN
+      value: cn=admin,dc=infra,dc=caasp,dc=local
+    - name: LDAP_BIND_PASSWORD
+      value: admin
```

Comment:
Is this something we can generate?

Reply:
Yep! That was the reason behind my email to caasp-devel :)